src/Controls/src/Core/Page/Page.cs

```diff
@@ -207,6 +210,17 @@ public bool IgnoresContainerArea
 		/// <inheritdoc/>
 		bool ISafeAreaView.IgnoreSafeArea => !On<PlatformConfiguration.iOS>().UsingSafeArea();
 
+#if IOS
+		/// <inheritdoc/>
+		bool IiOSPageSpecifics.IsHomeIndicatorAutoHidden => On<PlatformConfiguration.iOS>().PrefersHomeIndicatorAutoHidden();
```

Comment:
I'm wondering if we can leverage interfaces here a bit more to avoid having to always propagate the setting of this property down.

In theory we can check here if the parent has this set and then just return that from the interface. 

And then inside the xplat code you'd really just trigger handler updates and not have to do anything inside the renderers.

like, if the user sets this property on TabbedPage you just call "CurrentPage.Handler.Updatevalue"

could probably even just add that to the TabbedPage/Shell/FlyoutPage mappers